src/ctree-element-screen/ctree-segment-container.html

```diff
@@ -294,6 +298,7 @@
           component.set('data', variation.data);
           // TODO: don't set height percentage, let full image size be visible
           component.set('heightPercent', '56.25%');
+          this.$.component = this.getComponent();
```

Comment:
I'm not sure why `this.$.component` is set here.  Shouldn't it only be set when the component is created.